developer-docs-site/docs/nodes/ait/steps-in-ait3.md

```diff
@@ -48,7 +48,7 @@ To participate in the AIT-3 program, follow the below steps. Use these steps as
 
    1. Open the Aptos Wallet extension from the Extensions section of the Chrome browser, or by clicking on the puzzle piece on top right of the browser and selecting Aptos Wallet.
    2. Click **Create a new wallet**. 
-   3. When you are done creating the wallet, go to **Extensions** > **Aptos Wallet** and click on the gear icon on the bottom right. You will see the **Settings** screen. Click on the **Network** button and make sure the AIT-3 network is selected. 
+   3. Make sure to store your mnemonic somewhere safe. This account will be used in the future.
```

Comment:
Maybe use "seed phrase" instead of "mnemonic"? I think I remember seeing "seed phrase" in the Petra screen.